src/main/java/com/solbeg/Main.java

```diff
@@ -34,7 +37,9 @@ public Optional<User> findRichestUser() {
      * @return a list of Users
      */
     public List<User> findUsersByBirthdayMonth(Month birthdayMonth) {
-        throw new ExerciseNotCompletedException();
+        return users.stream()
+                .filter(user -> user.getBirthDay().getMonth().equals(birthdayMonth))
```

Comment:
can produce NPE if user,getBirthDay == null
so first you should add additional filtering for user.getBitrhDay
then it will be better to use at least reversed condition like birthdayMonth.equeals(user.getBirthDay().getMonth())
because if getMonth is null the it also can produce NPE